express/src/views/you-need-to-sign-in-again.njk

```diff
@@ -0,0 +1,12 @@
+{% extends "layout.njk" %}
+{% from "macros/error-summary.njk" import errorSummary %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+
+{% set active = "sign-in" %}
+
+{% block pageTitle %}Session Expired.{% endblock %}
+
+{% block mainContent %}
+  <h1 class="govuk-heading-l">Your session has expired, You need to sign in again</h1>
+  <p class="govuk-body"><a class="govuk-link" href="/sign-in">Sign-in</a>.</p>
+{% endblock %}
```

Comment:
This content doesn't seem correct, can you please check against the content designer's copy.

Reply:
Done.